frontend/src/components/Loader/Loader.tsx

```diff
@@ -1,12 +1,51 @@
 import { FormattedMessage } from 'react-intl';
-import { Dot } from './Loader.style';
+import { cn } from 'services/utils/cn';
 
 type LoaderProps = {
   className?: string;
   children?: React.ReactNode;
   loaded?: boolean;
 };
 
+const classNameAnimation = (index: number) => {
+  if (index === 0) {
+    return 'animation-delay-[-1.1s]';
+  }
+  if (index === 1) {
+    return 'rotate-[30deg] animation-delay-[-1s]';
+  }
+  if (index === 2) {
+    return 'rotate-[60deg] animation-delay-[-0.9s]';
+  }
+  if (index === 3) {
+    return 'rotate-[90deg] animation-delay-[-0.8s]';
+  }
+  if (index === 4) {
+    return 'rotate-[120deg] animation-delay-[-0.7s]';
+  }
+  if (index === 5) {
+    return 'rotate-[150deg] animation-delay-[-0.6s]';
+  }
+  if (index === 6) {
+    return 'rotate-[180deg] animation-delay-[-0.5s]';
+  }
+  if (index === 7) {
+    return 'rotate-[210deg] animation-delay-[-0.4s]';
+  }
+  if (index === 8) {
+    return 'rotate-[240deg] animation-delay-[-0.3s]';
+  }
+  if (index === 9) {
+    return 'rotate-[270deg] animation-delay-[-0.2s]';
+  }
+  if (index === 10) {
+    return 'rotate-[300deg] animation-delay-[-0.1s]';
+  }
+  if (index === 11) {
+    return 'rotate-[330deg]';
+  }
+};
```

Comment:
```suggestion
const classNameAnimation = index => {
  const delay = index < 11
    ? `animation-delay-[${(10 - index) / 10 - 0.1}]`
    : '';

  const rotate = index > 1
    ? `rotate-[${index * 30}deg]`
    : '';

  return `${rotate} ${delay}`;
};
```

```suggestion
const classNameAnimation = index => {
  const delay = `animation-delay-[${(10 - index) / 10 - 0.1}]`;
  const rotate = `rotate-[${index * 30}deg]`;

  if (index === 0) {
    return delay;
  }
  if (index === 11) {
    return rotate;
  }

  return `${rotate} ${delay}`;
};
```

```suggestion
const classNameAnimation = index => {
  const delay = `animation-delay-[${(10 - index) / 10 - 0.1}]`;
  const rotate = `rotate-[${index * 30}deg]`;

  switch (index) {
    case 0:
      return delay;
    case 1:
    case 2:
    case 3:
    case 4:
    case 5:
    case 6:
    case 7:
    case 8:
    case 9:
    case 10:
      return `${rotate} ${delay}`;
    case 11:
      return rotate;
  }
};
```

```suggestion
const classNameAnimation = index => {
  const delay = `animation-delay-[${(10 - index) / 10 - 0.1}]`;
  const rotate = `rotate-[${index * 30}deg]`;

  switch (index) {
    case 0:
      return delay;
    case 11:
      return rotate;
    default:
      return `${rotate} ${delay}`;
  }
};
```

Reply:
It should be fine to DRY these parts, but we need to keep the full class names because Tailwind doesn't understand interpolation and will purge them